jira/client.py

```diff
@@ -3157,3 +3188,13 @@ def __init__(self, options=None, basic_auth=None, oauth=None, async=None):
             "GreenHopper() class is deprecated, just use JIRA() instead.", DeprecationWarning)
         JIRA.__init__(
             self, options=options, basic_auth=basic_auth, oauth=oauth, async=async)
+
+
+def python_identifier(fieldstring):
+    return re.sub('[^0-9a-zA-Z_]', '', fieldstring.replace(' ', '_')).lower()
+
+def list_lookup(lst, key):
```

Comment:
Why not use dictionaries rather than lists and avoid having to do this iterative lookup?